content/en/docs/_index.md

```diff
@@ -29,5 +29,57 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
 ### Declarative UI controls
 Operators can behave like managed service providers. Their user interface on the command line are APIs. For graphical consoles OLM annotates those APIs with descriptors that drive the creation of rich interfaces and forms for users to interact with the Operator in a natural, cloud-like way. 
 
+
+## How OLM solves the cluster admin and developer needs.
+
+### Operator Installation and Management
+
+This packaging mechanism helps to simplify the multiple steps involved in deploying an Operator. OLM fulfills necessary metadata for visualizing them in compatible UIs, including installation instructions and API hints in the form of CRD descriptors. It helps cluster admin, developers to manage the lifecycle of all Operators and their associated services running across their clusters.
+
+### Dependency Resolution
+
+Using OLM eliminates the need for developers and cluster admins to have to worry about managing dependencies when developing one or more operators. OLM offers dependency resolution and upgrades the lifecycle of running operators. This functionality helps operator developers to resolve the problems related to dependency management.
+
+### Works similar to package Managers
+
+Operator developers may need to use one or more APIs for the operator installation and OLM helps solve this use case by verifying that all of the required APIs are available for the operator. OLM will attempt to find an operator that owns those missing requiredAPIs and install it. If there is no operator satisfying requiredAPIs, then OLM won't install the main operator.
+
+### Operator Groups
+
+To give cluster admins complete control over the permissions that OLM grants operators, OLM introduced the concept of `OperatorGroups`.
+
+* Defines the set of permissions that OLM may grant to member operators
+* Defines the set of namespaces that OLM may grant namespaced permissions in.
+
+## Building blocks of OLM
+
+### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)
+A `ClusterServiceVersion (CSV)` represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. 
```

Comment:
Minor nit: it's typically a markdown violation not having a newline between a header and the body.

```suggestion
### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)

A `ClusterServiceVersion (CSV)` represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. 
```

Reply:
We may also need to reword: `A `ClusterServiceVersion (CSV)` represents a particular version of a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc.` We may need to flush out this description, but I need to double-check if we already have documentation around these OLM custom resources.

Reply:
Sure @timflannagan - I will make newline between a header and body not only in above paragraph but also in rest of the newly added content.

Reply:
Modified according to suggestion. Please let me know about the reword of CSV defination.